mqtt/Dockerfile.arm

```diff
@@ -0,0 +1,12 @@
+FROM arm32v7/node:10.16.0-stretch-slim
+
+COPY qemu-arm-static /usr/bin
+
+COPY *.proto mqtt/* ./
+
+RUN npm rebuild && npm install
+
+EXPOSE 1883 8880
+
```

Comment:
Please remove `qemu-arm-static` at the end, to keep the image smaller.

Reply:
Removed, however new image have same size